machine/wnc_x86_64/wnc.patch/installer/x86_64/grub.d/50_onie_grub

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
Regarding setting --speed and --unit .  You do that in your machine.make file.  See the comments in https://github.com/opencomputeproject/onie/blob/master/machine/kvm_x86_64/machine.make about the `CONSOLE_SPEED`, `CONSOLE_DEV` parameters.